send_conn.go

```diff
@@ -3,12 +3,13 @@ package quic
 import (
 	"net"
 
+	"github.com/quic-go/quic-go/internal/protocol"
 	"github.com/quic-go/quic-go/internal/utils"
 )
 
 // A sendConn allows sending using a simple Write() on a non-connected packet conn.
 type sendConn interface {
-	Write(b []byte, gsoSize uint16) error
+	Write(b []byte, size uint16, ecn protocol.ECN) error
```

Comment:
Is this rename gsoSize => size required? I see the implementation uses the same name gsoSize.

Reply:
Keeping this `gsoSize` is definitely better.